Tests/v3LoadShedderIntegration/v3LoadShedderIntegrationTests/V3LoadShedderIntegrationTests.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  v3LoadShedderIntegrationTests.swift
+//
+//  Created by Andrés Boedo on 3/29/23.
+
+@testable import Purchases
+import StoreKit
+import StoreKitTest
+@testable import v3LoadShedderIntegration
```

Comment:
This also doesn't seem to follow convention of starting with uppercase? It might be ok though...